test/server/lib/validation/form_validators/ManageApplications.test.js

```diff
@@ -0,0 +1,149 @@
+const ManageApplication = require(`${serverRoot}/lib/validation/form_validators/ManageApplication`);
+const Validator = require(`${serverRoot}/lib/validation`);
+const logger = require(`${serverRoot}/config/winston`);
+const routeData = require(`${testRoot}/server/_fakes/data/routes/application`);
+const exceptions = require(`${testRoot}/server/_fakes/mocks`);
+let stubLogger;
+describe('server/lib/validation/applications', () => {
```

Comment:
The description for the test suite should try to describe the module under test as closely as possible. For example, here, I would have used: `describe('server/lib/validation/form_validatiors/ManageApplications', () => { ...`